### PR TITLE
fix (test): update EE test to provide more checking for April 24 logi…

### PR DESCRIPTION
**Pull request by ldimaggi**
…n workaround - runs locally, failing onCentos CI

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 536dd2330d71cc91c9b196ae7fc0235a3079ea14
  * Your subject line is longer than 50 characters
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/tests/work-item/work-item-list/EE/openshift-io-poc.spec.js [hunk 1]
```diff
@@ -59,11 +59,13 @@ describe('openshift.io End-to-End POC test - Scenario - New user registers', fun
...
-      if ( result ) {        
-        element(by.id("name")).click();
+      if ( result ) {       
+        browser.wait(until.elementToBeClickable(element(by.id("name")), constants.LONG_WAIT, 'Failed to find name/login name workaround')); 
+        element(by.id("name")).click();    
+        browser.wait(until.elementToBeClickable(element(by.xpath(".//*[@id='profilelink']/span")), constants.LONG_WAIT, 'Failed to find name/login profile workaround')); 
         element(by.xpath(".//*[@id='profilelink']/span")).click();
       } else {
-        //do nothing
+        console.log ("ERROR - April 24 workaround has failed");
```
Comment (hectorj2f) [inline src/tests/work-item/work-item-list/EE/openshift-io-poc.spec.js]: @ldimaggi shouldn't it be `console.log(` without a whitespace in between ?

Comment (ldimaggi) [inline src/tests/work-item/work-item-list/EE/openshift-io-poc.spec.js]: I don't see any whitespace - where do yo see this?

`console.log ("ERROR - April 24 workaround has failed");`

Comment (hectorj2f) [inline src/tests/work-item/work-item-list/EE/openshift-io-poc.spec.js]: ignore me! i need more sleep

**Pull request merged**